tests/console/clamav.pm

```diff
@@ -42,8 +42,11 @@ sub scan_and_parse {
 sub run {
     select_serial_terminal;
 
-    zypper_call('in clamav vim');
+    zypper_call('in clamav');
     zypper_call('info clamav');
+    # Create a random file to replace /usr/bin/vim
+    assert_script_run "dd if=/dev/urandom of=/usr/bin/vim bs=1M count=1";
```

Comment:
If vim is installed, this corrupts the binary. Also won't work if /usr is read-only.